functional-tests/src/test/groovy/specs/app/EditSystemPermissionsSpec.groovy

```diff
@@ -12,11 +12,8 @@ import spock.lang.Stepwise
 
 @Title("Functional tests for the EditSystemPermissions modal page")
 @Stepwise
-class EditSystemPermissionsSpec extends GebReportingSpec {
+class EditSystemPermissionsSpec extends LoggedInSpec {
   def setupSpec() {
```

Comment:
A bunch of files have empty setupSpec() methods that we can just remove.

Reply:
removed